ext/cfx-ui/src/app/servers/connecting-popup.component.ts

```diff
@@ -314,4 +315,10 @@ export class ConnectingPopupComponent implements OnInit {
 	doRetry() {
 		this.retry.emit();
 	}
+
+    onKeyPress(event: KeyboardEvent) {
+		if (this.closeKeys.includes(event.code)) {
```

Comment:
You shouldn't be using `event.code` as it ignores user layout and instead maps to the physical layout. Use `event.key` instead.